srcpkgs/ly/patches/config.ini.patch

```diff
@@ -0,0 +1,12 @@
+Change default tty to 7
+--- res/config.ini	2020-07-28 11:35:08.000000000 -0400
++++ "res/config copy.ini"	2020-08-02 21:37:52.572343702 -0400
+@@ -79,7 +79,7 @@
+ #term_reset_cmd = /usr/bin/tput reset
+ 
+ # tty in use
+-#tty = 2
++#tty = 7
+ 
+ # wayland setup command
+ #wayland_cmd = /etc/ly/wsetup.sh
```

Comment:
Why default to 7?

Reply:
7 is the tty that all (as far as I know) desktop environments launch on. With the previous default tty of 2, users would run into a scenario where they would log in via `ly` and receive no feedback that it had succeeded; they would need to manually switch to tty7 after logging in.

Reply:
I see. Could we simply uncomment this from the config file, to avoid patching the source as well? We should open an issue/PR with upstream about making it a build time option, as well.

Reply:
Yes, I can change the config file to avoid patching the source. Would the issue simply contain a request to add setting tty7 as the default as a build time option?

Reply:
I would make it generic, so any tty can be chosen as default.

Reply:
Issue opened: https://github.com/nullgemm/ly/issues/225